apps/site/assets/ts/schedule/components/schedule-finder/ServiceSelector.tsx

```diff
@@ -129,8 +188,9 @@ const ServiceSelector = ({
             className="schedule-finder__select"
             defaultValue={defaultServiceId}
             onChange={(): void => {
+              /* istanbul ignore next */
               if (ref && ref.current) {
```

Comment:
you might be able to do `ref.current!.value` (or something similar) if `current` is guaranteed to be set

Reply:
I'll give it a try, not familiar with the `!` operator in this context but I'm assuming from your comment that it's similar to Ruby's `try`?

Reply:
I think this is like a hint to the typescript interpreter in a case where, yes, technically it is possible for a value to be null in general, but in this specific logic branch it is guaranteed to have a value

Reply:
[googles a little] Interesting...so it's just to force certain behavior out of the type checker.

Reply:
@ryan-mahoney I think it might be possible for `current` not to be set, so I'll leave it.